app/(app)/(purchase-and-payment)/ticket-purchase.tsx

```diff
@@ -98,7 +98,7 @@ const TicketPurchasePage = () => {
         <ContentWithAvatar title={t('pendingTitle')} text={t('pendingText')}>
           <ActivityIndicator size="large" />
         </ContentWithAvatar>
-      ) : isError || data.paymentStatus === 'FAIL' ? (
+      ) : isError || data.paymentStatus === 'FAIL' || data.paymentStatus === 'ERROR' ? (
```

Comment:
This seems very dangerous to be easily forgotten by a developer when writing conditions.